### DISCUSSION TITLE
Download hangs with no output when given long .txt file

### DISCUSSION THREAD

**Discussion by Tylersuard** [kblin/ncbi-genome-download#204] [General]
Hello.  I used the method of creating a .txt file to designate which sequences to download, it had around 50k lines.  I ran the ncbi-genome-download -g command, and it hung for over an hour with no output.  I eventually just terminated it.

**Comment by kblin** [comment]
Depending on what you were trying to download, this could very easily happen. Running with the `-P` option to get a progress bar might give you some feedback on how long things might take.

To put things in perspective, I'm currently grabbing all bacterial RefSeq sequences of assembly level "complete, chromosome, and scaffold". That's 128,202 sequences, and just downloading the checksum files for these to figure out what data files I need to be downloading will take almost 2 hours on my machine/connection.
